Dockerfile

```diff
@@ -0,0 +1,145 @@
+FROM nvidia/cudagl:10.2-base-ubuntu18.04
+
+RUN apt-get update \
+    && apt-get upgrade -y \
+    && apt-get install -y \
+        git \
+        vim \
+        g++ \
+        gcc \
+        cmake
+
+WORKDIR /root
+
+
+RUN apt-get update --fix-missing \
+    && apt-get upgrade -y
+
+# Pangolin
+RUN git clone https://github.com/stevenlovegrove/Pangolin.git
+RUN apt-get install -y \
+    libgl1-mesa-dev \
+    libglew-dev \
+    ffmpeg \
+    libavcodec-dev \
+    libavutil-dev \
+    libavformat-dev \
+    libswscale-dev \
+    libavdevice-dev \
+    libdc1394-22-dev \
+    libraw1394-dev \
+    libjpeg-dev \
+    libpng-dev \
+    libtiff5-dev \
+    libopenexr-dev \
+    libeigen3-dev \
+    doxygen \
+    libpython3-all-dev \
+    libegl1-mesa-dev \
+    libwayland-dev \
+    libxkbcommon-dev \
+    wayland-protocols
+
+RUN apt-get install -y python3.7 python3-pip python3.7-dev
+RUN python3.7 -mpip install \
+    numpy \
+    pyopengl \
+    Pillow \
+    pybind11
+
+RUN cd Pangolin \
+    && git submodule init \
```

Comment:
Need to add a line saying `&& git checkout v0.6 \` after CDing into the Pangolin directory. Running this docker file with the newest version causes some problems with GL libraries.